### PR TITLE
Controlled vocabulary

### PR DESCRIPTION
**Pull request by smithjp**

### CODE REVIEW

File: lib/tasks/proquest_ingest.rake [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    resource.save
+
+    resource.label = work_attributes['title'][0]
+    resource.title = work_attributes['title']
+    resource.keyword =  work_attributes['keyword']
+    resource.degree_granting_institution = work_attributes['degree_granting_institution']
+    resource.abstract = [work_attributes['abstract']]
+    resource.advisor = work_attributes['advisor']
+    resource.degree = work_attributes['degree']
+    resource.academic_department = work_attributes['academic_department']
```
Comment (lfarrell) [inline lib/tasks/proquest_ingest.rake]: Hasn't this been replaced by affiliation?

File: lib/tasks/proquest_ingest.rake [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      genre = 'Dissertation'
+      resource_type = 'Dissertation'
+    else
+      genre = 'Thesis'
+      resource_type = 'Masters Thesis'
+    end
+
+    academic_concentration = metadata.xpath('//DISS_description/DISS_institution/DISS_inst_contact').text
+
+    affiliation = metadata.xpath('//DISS_description/DISS_institution/DISS_inst_contact').text.strip
```
Comment (lfarrell) [inline lib/tasks/proquest_ingest.rake]: Should this be in the work attributes?

File: lib/tasks/proquest_ingest.rake (hunk 1 above)
Comment (smithjp) [reply lib/tasks/proquest_ingest.rake]: Yes.  It looks like I missed a few.

File: spec/fixtures/authorities/biomed_departments.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## spec/fixtures/authorities/biomed_departments.yml
+terms:
+  - id: Biomedical Research Imaging Center, University of North Carolina at Chapel Hill
+    term: School of Medicine, Biomedical Research Imaging Center
+  - id: Biostatistics, Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC, 27599-7295, USA
```
Comment (bbpennel) [inline spec/fixtures/authorities/biomed_departments.yml]: In the test it looks like the full address as expressed in one of the submission documents is mapped as a term.  Don't see that mapping in the non-fixture versions though.  How will this vocabulary handle addresses in the production version?  The addresses are also generally not all that consistently formatted unfortunately.

Comment (smithjp) [reply spec/fixtures/authorities/biomed_departments.yml]: These examples were copied from the biomed_departments.yml authority file in the config/authorities directory.  I was not sure what the addresses from biomed would really look like, so I just copied the mappings from the google doc.  Do I need to include an address formatting service as well?

**Pull request merged**